tests/run.sh

```diff
@@ -22,5 +22,9 @@ virtualenv -p python2 venv
 . venv/bin/activate
 pip install -r requirements.txt
 
+echo "Begin clean up of environment"
```

Comment:
this removes possibility to run concurrent builds. But multiple tests are still possible.